scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/state/ECSState.java

```diff
@@ -16,5 +16,5 @@
 
 public interface ECSState {
 
-  ClusterSnapshot snapshotState(String clusterArn);
+  ClusterSnapshot snapshotState(String clusterName);
```

Comment:
This interface will have to change once we do a real job, to use assume role to get the customer cluster state. With only the cluster name, it's not possible to know customer service role arn. 

But as long as for M1, this PR should be fine to only focus on using new DS API.